src/processor/CodeSystemProcessor.ts

```diff
@@ -26,8 +33,8 @@ export class CodeSystemProcessor {
   }
 
   static process(input: any, fisher: utils.Fishable): ExportableCodeSystem {
-    // We need something to call the CodeSystem, so it must have a name or id
-    if (input.name != null || input.id != null) {
+    // It must be representable using the FSH CodeSystem syntax
+    if (this.isProcessableCodeSystem(input)) {
```

Comment:
This check is not strictly necessary anymore right? Because we only sent `ProcessableCodeSystems` into here from the `FHIRProcessor`? I don't really mind having the double checks, since I think if we wanted to get rid of them we'd have to do some refactoring to make the types work which may not be worth the benefit, mostly this is just a question for me to make sure I'm not missing something.

Reply:
Technically speaking, that is true, since we only call this from code that first filters out the unprocessable CodeSystems.  That said, since this is a public function, that could be called (in theory) from anywhere by anyone, I'd lean toward keeping the check.  As you note, it also helps w/ the typing.